user_docs/en/userGuide.t2t

```diff
@@ -347,9 +347,13 @@ The installed copy is also able to create a portable copy of itself at any time.
 The portable copy also has the ability to install itself on any computer at a later time.
 However, if you wish to copy NVDA onto read-only media such as a CD, you should just copy the download package.
 Running the portable version directly from read-only media is not supported at this time.
-Using the temporary copy of NVDA is also an option (e.g. for demonstration purposes), though starting NVDA in this way each time can become very time consuming.
 
-Apart from the inability to automatically start during and/or after log-on, the portable and temporary copies of NVDA also have the following restrictions:
+The [NVDA installer #StepsForRunningTheDownloadLauncher] can be used as a temporary copy of NVDA.
+Temporary copies prevent saving NVDA configuration such as settings or input gestures.

```

Comment:
@seanbudd, @Qchristensen, that is not true (at least for NVDA 2023.2beta1).
Indeed, modifying the gestures is allowed while running temp version of NVDA.

I think that the idea was to prevent writing on disk in temp versions only if it might cause issue with the installed version. E.g.: writing the config the config (settings) on the disk is not allowed because the config scheme (version) may be different and incompatible between installer and installed version. On the contrary writing gestures from a newer version does not cause any problem to older version: non-existing scripts are just silently ignored by the older version. If NVAccess changes the behaviour, it should be clarified and the reasons of it should be explained to developers.

Reply:
This has been fixed up in https://github.com/nvaccess/nvda/pull/15248

Note that the intention was actually to prevent any config from being saved to disk, as NVDA should be treated as a fully temporary copy in this case. This will be addressed in 2023.3 with more clear restrictions around secure screens, and temporary copies